router-map/data/tasks.py

```diff
@@ -81,12 +81,14 @@ def get_lldp_neighbour_details(self, interface):
             local_interface = neighbour.findtext('lldp-local-interface')
             parent_interface = neighbour.findtext('lldp-local-parent-interface-name')
             remote_chassis_id = neighbour.findtext('lldp-remote-chassis-id')
-            remote_interface_number = neighbour.findtext('lldp-remote-port-id')
+            remote_interface_id = neighbour.findtext('lldp-remote-port-id')
+            remote_interface_is_subtype = neighbour.findtext('lldp-remote-port-id-subtype')
             neighbour_details.append({
                 'local_interface': local_interface,
                 'parent_interface': None if parent_interface == '-' else parent_interface,
                 'remote_chassis_id': remote_chassis_id,
-                'remote_interface_number': remote_interface_number
+                'remote_interface': remote_interface_id,
+                'is_remote_interface_id_number': True if remote_interface_is_subtype == 'Locally assigned' else False
```

Comment:
There is a typo `remote_interface_is_subtype` -> `remote_interface_id_subtype`
and:
```suggestion
                'is_remote_interface_id_number': remote_interface_is_subtype == 'Locally assigned'
```